docs/add-ons/cert-manager.md

```diff
@@ -32,6 +32,10 @@ You can set an email address for expiration emails with:
 cert_manager_letsencrypt_email = "user@example.com"
 ```
 
+```
```

Comment:
we should add 1 sentence explanation what this is used for as an example, similar to the email address